www/_data/toc/de-dev-manual.yml

```diff
@@ -1,96 +1,138 @@
+        url: "guide/appdev/security/index.html"
+    -   name: "Whitelisting"
+        url: "guide/appdev/whitelist/index.html"
+    # -   name: "Publish"
+    #     url:
 -   name: "Create plugins"
     children:
-        -   name: "Create a plugin"
-            url: "guide/hybrid/plugins/index.html"
-        -   name: "Register with NPM"
-            url:
-        -   name: "Develop for platforms"
-            children:
-                -   name: "Android"
-                    url: "guide/platforms/android/plugin.html"
-                -   name: "Blackberry 10"
-                    url: "guide/platforms/blackberry10/plugin.html"
-                -   name: "iOS"
-                    url: "guide/platforms/ios/plugin.html"
-                -   name: "Windows"
-                    url: "guide/platforms/win8/plugin.html"
-                -   name: "WP8"
-                    url: "guide/platforms/wp8/plugin.html"
-        -   name: "Use Plugman"
-            url: "plugin_ref/plugman.html"
-        -   name: "Debug and test"
-            url:
+    -   name: "Create a plugin"
+        url: "guide/hybrid/plugins/index.html"
+    # -   name: "Register with NPM"
+    #     url:
+    -   name: "Develop for platforms"
+        children:
+        -   name: "Android"
+            url: "guide/platforms/android/plugin.html"
+        -   name: "Blackberry 10"
+            url: "guide/platforms/blackberry10/plugin.html"
+        -   name: "iOS"
+            url: "guide/platforms/ios/plugin.html"
+        -   name: "Windows"
+            url: "guide/platforms/win8/plugin.html"
+        -   name: "WP8"
+            url: "guide/platforms/wp8/plugin.html"
+    -   name: "Use Plugman"
+        url: "plugin_ref/plugman.html"
+    # -   name: "Debug and test"
+    #     url:
 -   name: "Advanced Topics"
     children:
-        -   name: "Embed Cordova in native apps"
-            url: "guide/hybrid/webviews/index.html"
-        -   name: "Pass data from Cordova app to host"
-            url:
-        -   name: "Next Steps"
-            url: "guide/next/index.html"
+    -   name: "Embed Cordova in native apps"
+        url: "guide/hybrid/webviews/index.html"
+    # -   name: "Pass data from Cordova app to host"
+    #     url:
+    -   name: "Next Steps"
+        url: "guide/next/index.html"
 -   name: "Reference"
     children:
-        -   name: "Config.xml"
-            url: "config_ref/index.html"
-        -   name: "Events"
-            url: "cordova/events/events.html"
-        -   name: "Plugin APIs"
-            url: "cordova/plugins/pluginapis.html"
-        -   name: "CLI"
-            url: "guide/cli/index.html"
-        -   name: "Hooks"
-            url: "guide/appdev/hooks/index.html"
-        -   name: "Plugin.xml"
-            url: "plugin_ref/spec.html"
+    -   name: "Config.xml"
+        url: "config_ref/index.html"
+    -   name: "Events"
+        url: "cordova/events/events.html"
+    -   name: "Plugin APIs"
+        children:
```

Comment:
Changed. I also reduced the padding in the ToC. Perhaps we should prioritize implementing the new ToC design, because it's getting a little unwieldy and the headings with children look really out of place.